optimum/neuron/utils/argument_utils.py

```diff
@@ -86,3 +86,28 @@ def expected_validation_function(arg):
                 "Tranium, set the environment variable OPTIMUM_DISABLE_STRICT_MODE to 1."
             )
             raise ValueError(f"{error_msg}\n{raise_error_msg}")
+
+
+def convert_neuronx_compiler_args_to_neuron(auto_cast="none", auto_cast_type=None, disable_fast_relayout=False):
```

Comment:
Let's add type annotations here because it can be ambiguous: `auto_cast` can be `None`, but also `"none"` for instance.

Reply:
Added.